app/src/main/java/com/github/dedis/student20_pop/model/Keys.java

```diff
@@ -1,28 +1,39 @@
 package com.github.dedis.student20_pop.model;
 
+import java.security.GeneralSecurityException;
 import java.util.Objects;
+import com.google.crypto.tink.subtle.Ed25519Sign;
+import com.google.crypto.tink.subtle.Hex;
+
 
 /**
  * Class modeling a set of Public and Private Keys
  */
 public final class Keys {
 
-    private final String publicKey;
-    private final String privateKey;
+    private String publicKey;
+    private String privateKey;
 
     /**
      * Constructor for the Keys, generates a set of public and private keys
      */
     public Keys() {
-        // Generate the public and private keys
-        this.publicKey = "";
-        this.privateKey = "";
```

Comment:
You should modify PersonTest because the class Person uses getPublicKey() and getPrivateKey()